operator/pkg/helm/helm.go

```diff
@@ -15,92 +15,59 @@
 package helm
 
 import (
+	"errors"
 	"fmt"
+	"io/fs"
 	"os"
 	"path/filepath"
 	"sort"
 	"strconv"
 	"strings"
 
 	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
 	"helm.sh/helm/v3/pkg/chartutil"
 	"helm.sh/helm/v3/pkg/engine"
 	"k8s.io/apimachinery/pkg/version"
-	"sigs.k8s.io/yaml"
 
 	"istio.io/istio/istioctl/pkg/install/k8sversion"
 	"istio.io/istio/manifests"
+	"istio.io/istio/operator/pkg/manifest"
 	"istio.io/istio/operator/pkg/util"
-	"istio.io/istio/pkg/log"
+	"istio.io/istio/operator/pkg/values"
+	"istio.io/istio/pkg/slices"
+	"istio.io/istio/pkg/test/util/yml"
 )
 
-const (
-	// YAMLSeparator is a separator for multi-document YAML files.
-	YAMLSeparator = "\n---\n"
-
-	// DefaultProfileString is the name of the default profile.
-	DefaultProfileString = "default"
-
-	// NotesFileNameSuffix is the file name suffix for helm notes.
-	// see https://helm.sh/docs/chart_template_guide/notes_files/
-	NotesFileNameSuffix = ".txt"
-)
+func Render(namespace string, directory string, iop values.Map) ([]manifest.Manifest, util.Errors, error) {
+	vals, _ := iop.GetPathMap("spec.values")
+	installPackagePath := iop.GetPathString("spec.installPackagePath")
+	f := manifests.BuiltinOrDir(installPackagePath)
```

Comment:
Looking at this makes me wonder if eventually it would be nice to let `istioctl` pull charts from a configured/named Helm repo like regular helm tooling as well.